<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>725</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>4199_Горшков Ал.О._Лабораторная работа №1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgba(220, 225, 230, 255),
        stop:0.3 rgba(235, 238, 242, 255),
        stop:0.7 rgba(235, 238, 242, 255),
        stop:1 rgba(220, 225, 230, 255)
    );
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame_lab_model_labels">
      <property name="minimumSize">
       <size>
        <width>954</width>
        <height>110</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: rgba(235, 238, 242, 180);
    border: 1px solid rgba(180, 185, 190, 200);
    border-radius: 8px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lab_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>930</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
    background-color: rgba(58, 65, 82, 200);
    color: #ffffff;
    border-radius: 6px;
    padding: 6px 12px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-weight: bold;
    font-size: 14pt;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>Лабораторная работа №1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="model_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>930</width>
           <height>34</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>34</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <italic>true</italic>
           <fontweight>Medium</fontweight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: rgba(200, 210, 220, 180);
    color: #102a43;
    border-radius: 4px;
    padding: 4px 8px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-weight: 500;
    font-size: 11pt;
}</string>
         </property>
         <property name="text">
          <string>Моделирование роста курса доллара США к рублю</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="data_label">
      <property name="minimumSize">
       <size>
        <width>556</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: rgba(58, 65, 82, 200);
    color: #ffffff;
    border: 1px solid rgba(47, 53, 66, 200); 
    border-radius: 4px;
    padding: 2px 6px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Данные из файла</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="regressions_label">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: rgba(58, 65, 82, 200);
    color: #ffffff;
    border: 1px solid rgba(47, 53, 66, 200); 
    border-radius: 4px;
    padding: 2px 6px;
}</string>
      </property>
      <property name="text">
       <string>Типы регрессий</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableView" name="data_tableView">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    background-color: #fdfdfd;
    alternate-background-color: #f0f4f8;
    gridline-color: #d1dbe5;
    font: 12pt &quot;Segoe UI&quot;;
    color: #102a43;
}

QHeaderView::section {
    background-color: #1d4ed8;
    color: white;
    padding: 4px;
    border: 1px solid #0f172a;
    font-weight: bold;
}

QTableView::item:selected {
    background-color: #93c5fd;
    color: #0c1a2c;
}

QTableView::item:hover {
    background-color: #dbeafe;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_linear_regr">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}
</string>
        </property>
        <property name="text">
         <string>Линейная регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_inverse_linear_regr">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string>Обратная линейная регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exp_regr">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string>Экспоненциальная регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_hyper_regr">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string>Гиперболическая регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_parab_regr">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string>Параболическая регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_log_regr">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string>Логарифмическая регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_power_regr">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string>Степенная регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_polynom_regr">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}

QPushButton:disabled {
    background-color: #E5E5E5;
    color: #7A7A7A;
    border: 1px solid #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string>Полиномиальная регрессия</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_ChooseFile">
      <property name="minimumSize">
       <size>
        <width>556</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}
</string>
      </property>
      <property name="text">
       <string>Выбрать .xlsx файл с исходными данными</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="choosenFile_label">
      <property name="minimumSize">
       <size>
        <width>391</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: rgba(58, 65, 82, 200);
    color: #ffffff;
    border: 1px solid rgba(47, 53, 66, 200); 
    border-radius: 4px;
    padding: 2px 6px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
